scripts/qt5_compile.sh

```diff
@@ -67,6 +67,7 @@ LINUX="
 "
 
 MACOS="
+  -debug-and-release \
```

Comment:
Is this flag necessary? From what I understand this will trigger two builds of QT (with/without extra debug configurations), this might be a bit of a shock for users just wanting a build of the VPN. Though perhaps it's targeted for internal builds.

Reply:
I prefer to keep `-release` only. Or have an extra param to pick debug/release/both.